core/templates/pages/exploration-editor-page/modal-templates/state-diff-modal.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="modal-header">
   <h3>
-    Card name: <[newStateName]>
-    <span ng-show="oldStateName">(was: <[oldStateName]>)</span>
+    Card name: {{newStateName}}
+    <span [hidden]="!oldStateName">(was: {{oldStateName}})</span>
```

Comment:
```suggestion
    <span [hidden]="!oldStateName">(was: {{ oldStateName }})</span>
```

Nit: add space after/before interpolation brackets.

Reply:
done